### DISCUSSION TITLE
Volume personalisation

### DISCUSSION THREAD

**Discussion by VillerotJustin** [lncrawl/lightnovel-crawler#2365] [Q&A]
Hi, I saw that a Volume is automatically generated every 100 chapter of the novel.

Is there is a way to say that volume 1 is from chapter 1 to 43, chapter 2 from 44 to 120, and so on ?

**Comment by Jack-Chronicle** [comment]
came here to see if there was a way myself... for now i'm just putting them all in one file

**Comment by thibaud200** [comment]
right now that possibility doen't exist in the crawler. my understanding is that the crawler bases his way of doing things on the information that are in the json files like the Volume numer and the Chapter content. So the mean to do that is to do a script that does a rework of the json file depending on the information provided for the arrangement you want. So it also means that the script must modify the metadata by collecting it as well

**Comment by thibaud200** [comment]
I've done a small python script to do just that since i was also in demand of one.
here is the file for those who need it[convertJson.zip](https://github.com/user-attachments/files/21347604/convertJson.zip) <<< you need to use the crawler to have the json files needed
